charts/quickwit/values.yaml

```diff
@@ -38,49 +38,55 @@ securityContext:
 
 # Additional global env
 environment: {}
-  # KEY: VALUE
+# KEY: VALUE
 environmentFrom: []
   # - secretRef:
   #     name: quickwit
   # - configMapRef:
-  #     name: quickwit
+#     name: quickwit
 
 configMaps: []
   # - name: configmap1
-  #   mountPath: /quickwit/configmaps/
+#   mountPath: /quickwit/configmaps/
 
 searcher:
   replicaCount: 3
 
   # Extra env for searcher
   extraEnv: {}
-    # KEY: VALUE
+  # KEY: VALUE
   extraEnvFrom: []
     # - secretRef:
     #     name: quickwit-searcher
     # - configMapRef:
-    #     name: quickwit-searcher
+  #     name: quickwit-searcher
```

Comment:
This comment shouldn't be changed as well

Reply:
Fixed